Add BBEs for the persistence feature (#4429)
ballerina-platform/ballerina-distribution: examples/persist-delete/persist_delete.md

# Persist write - Delete record

> The bal persist feature provide support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once model is defined, the client API is generated with resources based on the model. The generated

Reviewer: ```suggestion
The bal persist feature provides support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once the model is defined, the client API is generated with resources based on the model. The generated
```

> The Ballerina persistence feature provide support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once model is defined, the client API is generated with resources based on the model. The generated

Reviewer 2: ```suggestion
The Ballerina persistence feature provides support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once the model is defined, the client API is generated with resources based on the model. The generated
```

> The Ballerina persistence feature provide support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once model is defined, the client API is generated with resources based on the model. The generated

Reviewer 3: ```suggestion
The bal persist feature provide support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once model is defined, the client API is generated with resources based on the model. The generated
```

API can be used to query and manipulate the persistent data in the application.

> The generated client API provides `delete` resource method to delete record from the data store.

Reviewer: ```suggestion
The generated client API provides a `delete` resource method to delete records from the data store.
```

> The generated client API provides `delete` resource function to delete record from the data store.

Reviewer 2: ```suggestion
The generated client API provides a `delete` resource function to delete a record from the data store.
```

> The Ballerina persistence feature provide support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once model is defined, the client API is generated with resources based on the model. The generated
> API can be used to query and manipulate the persistent data in the application.
> The generated client API provides `delete` resource function to delete record from the data store.

Reviewer 2: ```suggestion
The Ballerina persistence feature provides support to manage data persistence in a Ballerina package. It starts with defining the application's data model. Once the model is defined, the client API is generated with resources based on the model. The generated
API can be used to query and manipulate the persistent data in the application.
The generated client API provides a `delete` resource function to delete a record from the data store.
```

> > **Note:** This example uses the Ballerina tables as the data store. You can MySQL and Google Sheets as the data store as well. For more information, see [Supported Data Stores](/learn/supported-data-stores/).

Reviewer 2: ```suggestion
> **Note:** This example uses Ballerina tables as the data store. You can use MySQL and Google Sheets as the data store as well. For more information, see [Supported Data Stores](/learn/supported-data-stores/).
```

#### Initialize the project
Execute the command below to initialize `bal persist` in the project.

::: out persist_init.out :::

#### Model the data

Add `Employee` entity with the following fields in the `model.bal` file inside the `persist` directory.

::: code persist_model.bal :::

#### Generate client APIs
Execute the command below to generate the Ballerina client API.

::: out persist_generate.out :::

#### Use the generated client API

> Using the generated client API, you can delete record from the datastore with the `delete` resource method.

Reviewer: ```suggestion
Using the generated client API, you can delete records from the datastore with the `delete` resource method.
```
